shared/constants/multichain/networks.ts

```diff
@@ -34,31 +39,45 @@ export type MultichainProviderConfig = ProviderConfigWithImageUrl & {
 export type MultichainNetworkIds = `${MultichainNetworks}`;
 
 export enum MultichainNetworks {
-  BITCOIN = 'bip122:000000000019d6689c085ae165831e93',
-  BITCOIN_TESTNET = 'bip122:000000000933ea01ad0ee984209779ba',
+  BITCOIN = BtcScope.Mainnet,
+  BITCOIN_TESTNET = BtcScope.Testnet,
+  BITCOIN_SIGNET = BtcScope.Signet,
 
-  SOLANA = 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp',
-  SOLANA_DEVNET = 'solana:EtWTRABZaYq6iMfeYKouRu166VU2xqa1',
-  SOLANA_TESTNET = 'solana:4uhcVJyU9pJkvQyS88uRDiswHXSCkY3z',
+  SOLANA = SolScope.Mainnet,
+  SOLANA_DEVNET = SolScope.Devnet,
+  SOLANA_TESTNET = SolScope.Testnet,
 }
 
+// TODO: This data should be provided by the snap
+export const NETWORK_TO_ACCOUNT_TYPE_MAP: Record<CaipChainId, string> = {
```

Comment:
This map and the `MULTICHAIN_NETWORK_TO_NICKNAME` seem to hold the same data. Can we consolidate these?

Reply:
`MULTICHAIN_NETWORK_TO_NICKNAME` will be removed

Reply:
There is still a mention, will get it out on the next refactor for block explorers